rector.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @file
+ * Rector configuration.
+ *
+ * Usage:
+ * ./vendor/bin/rector process .
+ *
+ * @see https://github.com/palantirnet/drupal-rector/blob/main/rector.php
+ */
+
+declare(strict_types=1);
+
+use Rector\CodeQuality\Rector\ClassMethod\InlineArrayReturnAssignRector;
+use Rector\CodeQuality\Rector\Empty_\SimplifyEmptyCheckOnEmptyArrayRector;
+use Rector\CodingStyle\Rector\ClassMethod\NewlineBeforeNewAssignSetRector;
+use Rector\CodingStyle\Rector\FuncCall\ArraySpreadInsteadOfArrayMergeRector;
+use Rector\CodingStyle\Rector\FuncCall\CountArrayToEmptyArrayComparisonRector;
+use Rector\CodingStyle\Rector\PostInc\PostIncDecToPreIncDecRector;
+use Rector\CodingStyle\Rector\Stmt\NewlineAfterStatementRector;
+use Rector\Config\RectorConfig;
+use Rector\DeadCode\Rector\If_\RemoveAlwaysTrueIfConditionRector;
+use Rector\Set\ValueObject\SetList;
+use Rector\Strict\Rector\Empty_\DisallowedEmptyRuleFixerRector;
+
+return static function (RectorConfig $rectorConfig): void {
+  $rectorConfig->paths([
+    __DIR__ . '/**',
+  ]);
+
+  $rectorConfig->sets([
+    SetList::PHP_80,
+    SetList::PHP_81,
+    SetList::CODE_QUALITY,
+    SetList::CODING_STYLE,
+    SetList::DEAD_CODE,
+    SetList::INSTANCEOF,
+    SetList::TYPE_DECLARATION,
+  ]);
+
+  $rectorConfig->skip([
+    // Rules added by Rector's rule sets.
+    ArraySpreadInsteadOfArrayMergeRector::class,
+    CountArrayToEmptyArrayComparisonRector::class,
+    DisallowedEmptyRuleFixerRector::class,
+    InlineArrayReturnAssignRector::class,
+    NewlineAfterStatementRector::class,
+    NewlineBeforeNewAssignSetRector::class,
+    PostIncDecToPreIncDecRector::class,
+    RemoveAlwaysTrueIfConditionRector::class,
+    SimplifyEmptyCheckOnEmptyArrayRector::class,
+    // Dependencies.
+    '*/vendor/*',
+    '*/node_modules/*',
+    '*/tests/*',
```

Comment:
@tannguyen04 
Please remove this and update code in tests.

Reply:
Updated.